_pages/en/strategy-build-vs-buy.md

```diff
@@ -37,66 +44,140 @@ th {
+The benefits to increasing the flexibility and interoperability of IT Solutions architectures as well as reusability and scalability of its IT Products are expected to reduce risk [^1], reduce technical debt over time, increase client satisfaction, and increase overall confidence in the department and its staff.
+
+This strategy capitalizes on existing IT initiatives (such as the [IITB Way Forward](http://dialogue/grp/IITB-DGIIT-Gov-New-Nouveau/Documents/Departmental%20IMIT%20Plan/19-20%20Plans/IITB%20Moving%20Forward%20v2.docx)) to compliment them by adding attention to them, as well as complimenting them with new activities.
+
+This strategy's goal is to define a build versus buy framework, and provide a roadmap in getting to the adoption of this framework.
+
+More details in [Annex A](#appendix-a---business-case-diagnostic).
+
+## Guiding Policy
+
+The following policy reflects the decision adopted by the **CIO of ESDC** (approval by CIO not yet obtained) when deciding whether an IT solution should be built or bought.
+Each policy statement is a declaration of that decision and has received the endorsement of its associated area of governance body (endorsements not yet obtained, see section [coherent set of actions](#coherent-set-of-actions)).
+
+This policy becomes active when IT Solutions are to be obtained, whether delivered by IITB or by a third party provider.
+Once active, all teams involved in the project, and the IT products involved in the IT solution, must comply with this guiding policy.
+
+This Guiding Policy has been prepared by taking into consideration alignment and compliance with existing policy instruments and does not replace them.
+Stakeholders are expected to still comply with existing policy instruments including, but not limited to:
+
+- ESDC [Policy on Project and Programme Management (PPPM)](https://gpp-ppm.service.gc.ca/sites/pwa/ESDCKnowledgeRepository/All%20Documents/Policy%20on%20Project%20and%20Programme%20Management.pdf)
+- ESDC [Directive on Project Management](https://gpp-ppm.service.gc.ca/sites/pwa/ESDCKnowledgeRepository/All%20Documents/Policy%20on%20Project%20and%20Programme%20Management.pdf)
+- ESDC Information Management Policy (being drafted)
+- ESDC [Procurement Policies](http://iservice.prv/eng/finance/purchasing/policy.shtml)
+- ESDC Security Policy (being drafted)
+- TBS [Directive on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32601)
+- TBS [Directive on Security Management](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32611)
+
+### Governance, Compliance, and Reporting
+
+1. IT Solutions are understood as a collection of IT Products and as such, each IT Product must be assessed on its own as well as the sum of all the IT Products composing the IT Solution.
+2. IT Solutions are obtained in a way that minimizes the risks to the department and to its ability to Canadians (see [Risk Managing the Obtention of Software](#risk-managing-the-obtention-of-software)). As such:
```

Comment:
IT Solutions are obtained in a way that minimizes the risks to the department and to its ability to Canadians (see [Risk Managing the Obtention of Software](#risk-managing-the-obtention-of-software)). As such:
=>
IT Solutions are obtained in a way that minimizes the risks to the department and to its ability to *serve* Canadians (see [Risk Managing the Obtention of Software](#risk-managing-the-obtention-of-software)).

Reply:
Oh, thanks, typo!